src/test/integration/services/nft.e2e-spec.ts

```diff
@@ -381,6 +392,14 @@ describe('Nft Service', () => {
       expect(count).toStrictEqual(10000);
     });
 
+    it("should return total number of NFTs that contains nonce equal with 200 ", async () => {
```

Comment:
I think this test is not accurate enough, maybe extend it with before & after to know the exact count of nfts.
Also, a test scenario with nonce for collection nfts would work.

Reply:
Test was refactored with after timestamp filter